specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/azureADMetrics.json

```diff
@@ -0,0 +1,443 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.aadiam/azureADMetrics": {
+      "get": {
+        "tags": [
+          "azureADMetrics"
+        ],
+        "operationId": "azureADMetrics_List",
+        "x-ms-examples": {
+          "azureADMetricsGetList": {
+            "$ref": "./examples/AzureADMetricsList.json"
+          }
+        },
+        "description": "Operation to return the list of AzureADMetrics resources scoped to the resourceGroup.",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../2020-03-01-preview/privateLinkForAzureAD.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get list of AzureADMetrics resources For AzureAD.",
+            "schema": {
+              "$ref": "#/definitions/azureADMetricsListResult"
+            }
+          },
+          "default": {
+            "description": "Error, failed to get list of AzureADMetrics resources.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "azureADMetricsListResult": {
+      "description": "A list of AzureADMetrics resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of AzureADMetrics resources.",
+          "items": {
+            "$ref": "#/definitions/azureADMetricsConfig"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "azureADMetricsConfig": {
+      "description": "AzureADMetrics resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "ID of the AzureADMetrics configuration object.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the AzureADMetrics configuration object.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the AzureADMetrics configuration object.",
+          "type": "string"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AzureResourceBase"
```

Comment:
>AzureResourceBase [](start = 33, length = 17)

can you use the ProxyResource definition in specification\common-types\resource-management\v1\types.json instead?  "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"

Reply:
Sorry, this should be the TrackedResource definition, not Proxy since you have location and tags (missing from the current spec)

---
In reply to: [436102722](https://github.com/Azure/azure-rest-api-specs/pull/9636#discussion_r436102722) [](ancestors = 436102722)

Reply:
After I addressed your changes and changing to TrackedResource, I am getting errors about missing required property: location which I have not gotten before, as well as additional validation errors for the examples. Could you help me figure these errors out? There are no errors when #/definitions/AzureResourceBase is used.

Reply:
The examples need to be updated to include "location" and "tags".  These are properties that all tracked resources should support.  Your old definition was missing those properties.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md

Reply:
Thank you!